Update patterns.h

Lynxxde opened the pull request:
Code reworked

Lynxxde committed:
Update patterns.h

Code reworked

dga02155 commented:
This is much more readable. I figured out what was going on with the previous version, but wish I had started with this.
:+1:

Lynxxde commented:
Thanks, i'm happy someone is still alive here. :smile:

durkie commented:
Hi, sorry! I'm glad you folks are here too. :)

I had been hoping to review both of your PRs together Holger since you can't really have one without the other. They are without a doubt easier to read, I just wanted to give a bit more thought on naming conventions and keeping overall sketch size in check. Gimme just a little more time!

Lynxxde committed:
Update Mod-Sketch.ino

ew from v0.2.2:
G = get Powerlevel : Returns Powerlevel
v = get Version : Returns Version information

New from v0.2.4:
U ... = Play the User transmitted pattern, for example:
U 3,255 0 0 500, 0 0 255 500, 0 255 0 500
3 Entries
First Motor 255, Second 0, Third 0, for 500ms
First Motor 0, Second 0, Third 255, for 500ms
First Motor 0, Second 255, Third 0, for 500ms

allStop Pattern added

New from v0.2.5:
u = Returns current Userpattern

--------------
Changed Versionstring to "Mod-Sketch v0.2.5"

Lynxxde committed:
Update README.md

Added all changes in newer versions

Lynxxde committed:
Update README.md

Lynxxde committed:
Update README.md

Lynxxde committed:
Update README.md

Added all changes in newer Versions
